app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -488,4 +490,18 @@ object PatientMocker {
         businessIds = emptyList()
     )
   }
+
+  fun country(
+      isoCountryCode: String = "IN",
+      endpoint: String = "https://simple.org",
+      displayName: String = "India",
+      isdCode: String = "91"
+  ): Country {
+    return Country(
+        isoCountryCode = isoCountryCode,
+        endpoint = URI.create(endpoint),
```

Comment:
I am slightly concerned about this, because `country` is no longer a pure function, because `URI.create(String)` can throw exceptions 😕

Reply:
This is intentional. We do not want an invalid endpoint to be provided. In production, this is something we control so given an invalid endpoint should be a fatal error.

Reply:
Should the app crash in case of a misconfiguration?

Reply:
I am of the opinion that it should.

Reply:
Discussed offline: we'll retain the `URI.create()` for now and see if you need to change this later since Retrofit will anyway throw if the url is malformed.